README.md

```diff
@@ -38,6 +38,45 @@ assert s == 10
 ```
 Note: here nimpy relies on your local python installation.
 
+
+## Importing Nim Extensions Directly
+
+For a convenient way to import your Nim extension modules directly, you can use
+[Nimporter](https://github.com/Pebaz/Nimporter). It scans Python's path to find
```

Comment:
Lets leave the first sentence, and remove the rest. No need to duplicate your README.